Sources/SwiftGRPC/Core/Channel.swift

```diff
@@ -40,12 +40,15 @@ public class Channel {
   ///
   /// - Parameter address: the address of the server to be called
   /// - Parameter secure: if true, use TLS
-  public init(address: String, secure: Bool = true) {
+  /// - Parameter args: list of arguments
+  public init(address: String, secure: Bool = true, args: [Arg] = []) {
     host = address
+    var cargs = args.map({ $0.toCArg() })
```

Comment:
In case the corresponding argument to the C function can be made `const` (see above), we might be albe to change this into `let`.

Also, would you mind using the trailing closure without parentheses?